apps/api/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": ["../../.eslintrc.js"],
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
+      "rules": {}
+    },
+    {
+      "files": ["*.ts", "*.tsx"],
+      "rules": {}
+    },
+    {
+      "files": ["*.js", "*.jsx"],
+      "rules": {}
+    }
```

Comment:
The second and third objects here seem duplicative. Is there a reason they need to be re-declared?

Reply:
Cool, fine with leaving them. If it's generated, there might be a reason.